cm/adapters/outbound/akafka.py

```diff
@@ -51,8 +52,12 @@ async def publish_sample_updated(
         self, *, sample_no_auth: models.SampleNoAuth
     ) -> None:
         """Publish an event saying that a sample was updated"""
+        payload = sample_no_auth.dict()
+        for key, value in payload.items():
+            if isinstance(value, datetime):
+                payload[key] = str(value)  # maintain ignorance of specific fields
```

Comment:
To get a JSON-compatible dict you can do something like this:

```suggestion
        payload = json.loads(sample_no_auth.json())
```

Reply:
Yep, that worked. Thanks